contracts/RFQ.sol

```diff
@@ -174,4 +179,25 @@ contract RFQ is IRFQ, Ownable, TokenCollector, EIP712 {
             fee
         );
     }
+
+    // Only used when taker token is ETH
+    function _collecETHAndSend(address to, uint256 amount, bytes calldata data, bool makerReceivesWETH) internal {
+        if (makerReceivesWETH) {
+            weth.deposit{ value: amount }();
+            weth.transfer(to, amount); // NOTE: data not used, is this an issue?
```

Comment:
[Question]: Is directly sending WETH using `weth.transfer` ok? I initially tried to use `_collect` here, but eventually think its kinda unnecessary since we are transferring from contract to maker.